src/level_set/FESurfaceDistanceEvaluator.cpp

```diff
@@ -709,6 +913,21 @@ FESurfaceDistanceEvaluator::getClosestPoint3D(const IBTK::Vector3d& P,
     double s = b * e - c * d;
     double t = b * d - a * e;
 
```

Comment:
Lines 

https://github.com/IBAMR/IBAMR/blob/ramakrishnan/fe-surface-distance-evaluator/src/level_set/FESurfaceDistanceEvaluator.cpp#L897-L900

We should use `N0, N1, N2` instead of `n0,n1,n2` here:

```
    const libMesh::Point& N0 = elem->point(0);
    const libMesh::Point& N1 = elem->point(1);
    const libMesh::Point& N2 = elem->point(2);
    const IBTK::Vector3d B(N0(0), N0(1), N0(2));
    const IBTK::Vector3d D = B - P;

    const IBTK::Vector3d E0(MAKE_EDGE(B, N1));
    const IBTK::Vector3d E1(MAKE_EDGE(B, N2));
```

as we use `n0,n1,n2` later down to compute nodes of `nbr_elem` nodal points.

Reply:
How are `N0`, `N1`, `N2` related to `n0`, `n1`, `n2`?

In general I would recommend avoiding having variable names differ only by capitalization; it becomes easy to make mistakes. One exception is for cases like nonlinear mechanics where you might indicate corresponding quantities in the reference and current configurations by switching between uppercase/lowercase.

Reply:
`N0`, `N1`, `N2` are current element's node pointers and  `n0`, `n1`, `n2` are its neighboring element's pointer. Earlier both used the same variable name, which could also lead to a mistake in the future. 

How about using `elem_n0`, `elem_n1`, `elem_n2` instead?

Reply:
Sure. Maybe prefix the "neighbor" ones instead of the "local" ones?

Reply:
Agreed. @Raamkrishnan : Let's use `nbr_n0`,`nbr_n1`, `nbr_n1` at that place along with the prior suggestion.

Reply:
Hi, I incorporated the changes and updated it.